memory-bank/02-build-system.md

```diff
@@ -0,0 +1,96 @@
+# Apache Traffic Server - Build System & Configuration
+
+## Build System & Configuration
+
+### CMake Build System (Version 10+)
+```bash
+# Basic build
+cmake -B build
+cmake --build build
```

Comment:
I would just have one build command. Would be nice to have a command box with a copy command.